package.json

```diff
@@ -46,8 +47,8 @@
       "https://mail.google.com/*",
       "https://docs.google.com/*",
       "https://encrypted.google.com/*",
-      "http://mxr.mozilla.org/*",
-      "http://w3c*.github.io/*"
+      "https://*.101weiqi.com/*",
```

Comment:
Why did you replace `http://mxr.mozilla.org/*` with `https://*.101weiqi.com/*`?

Reply:
https://mxr.mozilla.org/ still exist?
101weiqi: #42

Reply:
Shouldn't it be `https://101weiqi.com/guan/*`?

Reply:
Break others, for example: https://101weiqi.com/q/14910/